### ISSUE TITLE
ipv6

### ISSUE THREAD

**Issue by mikkelke** [alajmo/sake#9]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

When entering a ipv6 address i get the following error:
dial tcp: address fd58:9374:59b8:430c:8299:936b:7f1:49c:22: too many colons in address
If I put square brackets around i get:
error: cannot unmarshal !!seq into string

**Describe the solution you'd like**

I would like to be able to connect to servers using ipv6

**Additional context**

**Comment by mikkelke** [comment]
Hi alajmo.
Thanks for the quick respond and for fixing the problem.

Best regards, 
Mikkel Eskildsen

**Comment by alajmo** [comment]
Thank you for your kind words, I've released 0.1.8 that includes the IP6 fix. Let me know if works

**Comment by mikkelke** [comment]
I can connect to IPv6 and it works but every time i run a command eg.
sake exec --all --parallel 'hostname'

Then I need to confirm the host identity for all of the host everytime i run it:
Unknown Host: [fd58:9374:.....
Fingerprint: SHA256:.......
Would you like to add it? type (y)es or (n)o: y

I have connected directly with ssh to the host and added the identity to /.ssh/known_hosts

How can I make it remember the host identity?

**Comment by alajmo** [comment]
Thanks for reporting, I found the issue, it seems it may be an issue with the go  std library `golang.org/x/crypto/ssh/knownhosts`.
When you use ip6 with a port number, you must prefix the IP address with brackets (to distinguish from the separator used in ip6 colon `:`).

Apparently, when they serialize the ip6 address, they append the square brackets:

```
[2001:3984:3989::10] ecdsa-sha2-nistp256 xxx
```

But then when you use their method `knownhosts.New(knownFile)` you get the error:

```
knownhosts: /home/samir/projects/sake/test/playground/known_hosts:4: address [2001:3984:3989::10]: missing port in address
```

So I'll circumvent their method and remove the brackets from the known_hosts file and push up a fix.

In the meantime, if you want, you can disable known hosts verification by adding `disable_verify_host: true` at the root level of your `sake` config. Or, add your ip6 address directly to the `known_hosts` file like this

```
2001:3984:3989::10 ecdsa-sha2-nistp256 xxx
```

Currently it gets saved as this:

```
[2001:3984:3989::10] ecdsa-sha2-nistp256 xxx
```

**Comment by mikkelke** [comment]
Thanks for the quick reply. 
For now i use disable_verify_host: true and everything works then.

**Comment by alajmo** [comment]
This should be fixed now with release v0.10.1. Please re-open the issue if it persists.
